sadcowpng/sharedspaces/views.py

```diff
@@ -0,0 +1,18 @@
+from django.shortcuts import render
+
+from django.http import HttpResponse
```

Comment:
nit: not specified by PEP8, but generally accepted best-practices have imports places together sorted alphabetically. pep8 specifies two lines below import statements

Reply:
Not updated. Just following PEP8 standards.